flakeheaven/_logic/_snapshot.py

```diff
@@ -12,23 +14,23 @@
 
 
 CACHE_PATH = Path(
-    os.environ.get('FLAKEHEAVEN_CACHE', Path.home() / '.cache/flakeheaven'),
+    os.environ.get('FLAKEHEAVEN_CACHE', Path().resolve() / '.cache/flakeheaven'),
 )
 
-THRESHOLD = int(os.getenv('FLAKEHEAVEN_CACHE_TIMEOUT', 3600 * 24))  # default is 1 day
+THRESHOLD = int(os.getenv('FLAKEHEAVEN_CACHE_TIMEOUT', timedelta(days=1).seconds))
```

Comment:
If you are going to change it, make sure the outcome is the same. I think the proper answer should be `timedelta(days=1).total_seconds()` instead?